src/korean.js

```diff
@@ -21,13 +21,21 @@ class Korean {
     if (word[wordLength - 2] === '하') {
         conjugate = word.slice(0, wordLength - 2);
         return conjugate.concat('해');
-    } else {
+    } else if (word[wordLength -2] === '르') {
```

Comment:
I see, there could be other vowels that are not ㅗ or ㅜ following the 르 ? Probably the irregulars need to be caught first because they do seem to have more conditions in order to conjugate them. I did change the korean-test to make a separate cateogry for irregular korean present tense conjugations in my PR (an idea to single out the irregulars). There looks like there are more irregulars here, according to this [article](https://www.howtostudykorean.com/unit1/unit-1-lessons-1-8/unit-1-lesson-7/#%E3%84%B9i).